src/templates/react/Content/src/State.fs

```diff
@@ -0,0 +1,52 @@
+module State.App
+
+open Elmish
+open Elmish.Browser.Navigation
+open Elmish.Browser.UrlParser
+open Fable.Import.Browser
+open Global
+open Types.App
+
+let pageParser: Parser<Page->Page,Page> =
+  oneOf [
+    map About (s "about")
+    map Counter (s "counter")
+    map Home (s "home")
+  ]
+
+let urlUpdate (result: Option<Page>) model =
+  match result with
+  | None ->
+    console.error("Error parsing url")
+    ( model,Navigation.modifyUrl (toHash model.currentPage) )
+  | Some page ->
+      match page with
+      | Counter ->
+          let (counter, counterCmd) = State.Counter.init ()
```

Comment:
Oh ok I see.

The missing part in my mind was:

```fs
 let model,cmd =
       urlUpdate result
         { page = Home
           ...
```

Will update the template this evening with it.